server/export.go

```diff
@@ -43,3 +60,12 @@ func ExportCmd(ctx *Context, cdc *wire.Codec, appExporter AppExporter) *cobra.Co
 		},
 	}
 }
+
+func isEmptyState(home string) (bool, error) {
```

Comment:
@ValarDragon is this sanity check enough or is there a more direct method we can perform via some attempted state lookup?

Reply:
Seems fine to me since we ensure all file writes persist to disk. I think we can merge this, and then write an issue to query state directly later. (#postlaunch concern)